proto/src/main/proto/stations.proto

```diff
@@ -22,11 +25,16 @@ message StationDb {
     map<uint32, Line> lines = 4;
 
     // Provides a hint for LocaleSpan about which language to mark up the
-    // names with, to improve pronunciation.
+    // English names with, to improve pronunciation.
     string tts_hint_language = 5;
 
     // Licensing notice for the file.
     string license_notice = 6;
+
+    /* A map of language codes (two-letter ISO code) to language ID, local to
+     * this file, used for `Names.other` and `Names.other_short`.
+     */
+    map<string, uint32> languages = 7;
```

Comment:
I think we need more than just id. I think it needs to map to a message containing ID but also e.g. which is primary local language. E.g. on English system if the checkbox "show local language" is on and we scan Rav-Kav then Hebrew will be the language to show. Other attributes might include tts hints. Hebrew omits vowels and I can imagine having 2 IDs for Hebrew: first one is used for display purposes (E.g for Haifa: חיפה‎) and second one with vowels (חֵיפָה‎) is retrieved but not shown and instead is stuffed into tts hints. For this we need a second field in language message which species the key for tts hints.